builtin/client/chatcommands.lua

```diff
@@ -0,0 +1,28 @@
+-- Minetest: builtin/client/chatcommands.lua
+
+
+core.register_on_sending_chat_messages(function(message)
+	if not (message:sub(1,1) == "/") then
```

Comment:
this uses a few functions that aren't available on the server and it removes parts of the server sided version of this function that aren't need and it also takes  a different number of arguments.